Makefile.in

```diff
@@ -350,7 +350,7 @@ install: install-h install-lib install-pc @INSTALL_CONFIG@ install-manpages $(py
 
 uninstall: uninstall-manpages @UNINSTALL_CONFIG@ uninstall-h uninstall-lib uninstall-pc $(pyldns_uninst) $(pyldnsx_uninst) @UNINSTALL_P5_DNS_LDNS@ @UNINSTALL_DRILL@ @UNINSTALL_EXAMPLES@
 
-destclean: uninstall
+distclean: uninstall
```

Comment:
`distclean` is traditional, but obviously this change will break anyone using the old name.

Reply:
I doubt anyone will use that except developers, so I'm fine with the rename.